### ISSUE TITLE
Error with confirm module

### ISSUE THREAD

**Issue by dokuzbir** [danschultzer/coherence_assent#4]
My app doesnt have coherence confirm module. I get that error

```
Coherence.User.confirmed?/1 is undefined or private. Did you mean one of:

      * confirmable?/0
```

**Comment by danschultzer** [comment]
Can you use PR #4 as dependency and see if it now works for you?

**Comment by dokuzbir** [comment]
yes it works.

 I dont know maybe you didnt add fpr a reason.  Can you also add to schema that `on_delete: :delete_all` please.

**Comment by danschultzer** [comment]
> I dont know maybe you didnt add for a reason. Can you also add to schema that on_delete: :delete_all please.

That makes sense. I think it wasn't added to keep it simple, but I agree that it should be on by default.

**Comment by danschultzer** [comment]
I've released `v0.2.4` to fix the issues!
